RepoDb.Core/RepoDb/Extensions/PropertyInfoExtension.cs

```diff
@@ -80,8 +81,10 @@ public static IEnumerable<PropertyInfo> AsEnumerable(this PropertyInfo property)
         /// <returns>A string containing the mapped name.</returns>
         public static string GetMappedName(this PropertyInfo property)
         {
-            var attribute = (MapAttribute)GetCustomAttribute(property, typeof(MapAttribute));
-            return attribute?.Name ??
+            var columnAttribute = (ColumnAttribute) GetCustomAttribute(property, typeof(ColumnAttribute));
```

Comment:
And also here, `MapAttribute` must be in higher priority/precedence than `ColumnAttribute`?